engines/instance_verification/app/controllers/instance_verification/billing_check_controller.rb

```diff
@@ -0,0 +1,29 @@
+module InstanceVerification
+  class BillingCheckController < ::ApplicationController
```

Comment:
Why this nested path, and not `engines/instance_verification/app/controllers/instance_verification_controller.rb`?